.github/workflows/test.yml

```diff
@@ -0,0 +1,53 @@
+name: Run tests
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+jobs:
+    test:
+      runs-on: ubuntu-latest
+      strategy:
+        matrix:
+          python: [2.7, 3.6, 3.7, 3.8, 3.9]
+      steps:
+        - uses: actions/checkout@v2
+        - name: Setup Python
+          uses: actions/setup-python@v2
+          with:
+            python-version: ${{ matrix.python }}
+        - name: Install Tox and any other packages
+          run: |
+            python -m pip install --upgrade pip
+            pip install tox tox-gh-actions
+        - name: Run Tox
+          # Run tox using the version of Python in `PATH`
+          run: tox -epy
+    flake8:
+      runs-on: ubuntu-latest
+      steps:
+        - uses: actions/checkout@v2
+        - name: Setup Python
+          uses: actions/setup-python@v2
+          with:
+            python-version: 3.9
```

Comment:
So we don't want to run flake the matrix of versions?

Reply:
This doesn't seem useful for flake8 since linting isn't version dependent?

Reply:
Sure, I can see that